packages/block-directory/package.json

```diff
@@ -20,6 +20,10 @@
 	"main": "build/index.js",
 	"module": "build-module/index.js",
 	"react-native": "src/index",
+	"sideEffects": [
+		"build-style/**",
```

Comment:
This line is redundant, since the following one would already match these paths, but I feel it's best to be explicit about styles, in its own rule. Let me know if you disagree and would like me to remove these redundant lines.

Reply:
It's up to you, you are correct that it could be removed.